faker/providers/internet/__init__.py

```diff
@@ -165,8 +163,12 @@ def domain_word(self,):
     def tld(self):
         return self.random_element(self.tlds)
 
-    def url(self):
-        pattern = self.random_element(self.url_formats)
+    def url(self, schemes=None):
+        if not schemes:
```

Comment:
use `if schemes is None:`. This way, users can pass an empty list and have schemeless URL ('malformed' urls like that are often used in HTML pages to load a resource regardless of http/https)

Reply:
In this case would it be something like `www.domain.com`, and not `://www.domain.com`, i suppose

Reply:
No, the expected output for `schemes=[]` would be a protocol-relative URL like `://www.domain.com`

Reply:
Done :+1: I added a bit of documentation too

Reply:
I don't understand how coveralls check had failed, it shows decreased coverage for parts I've not touched or whatever